addons/point_of_sale/static/src/js/ChromeWidgets/CashierName.js

```diff
@@ -15,6 +15,9 @@ odoo.define('point_of_sale.CashierName', function(require) {
             const id = user_id ? user_id : -1;
             return `/web/image/res.users/${id}/avatar_128`;
         }
+        get class() {
+            return {'not-clickable': true};
```

Comment:
Why do you need 'not-clickable' and 'oe-status' can't you have only one class, so one style without the class and another one with the class?

Reply:
"not-clickable" remove the highlight effect when hovering because the cashier name is inside the `status-button` that has highlight effect. `oe_status` is adding the cursor pointer